engines/optix/OptiXModel.h

```diff
@@ -114,6 +121,19 @@ class OptiXModel : public Model
     std::map<std::string, optix::Buffer> _optixTextures;
     std::map<std::string, optix::TextureSampler> _optixTextureSamplers;
 
+    // Transfer function
+    struct OptiXTransferFunction
+    {
+        optix::Buffer colors{nullptr};         // vec3fs
+        optix::Buffer opacities{nullptr};      // floats
+        optix::Buffer simulationData{nullptr}; // floats
+        bool initialized{false};
+    };
+
+    OptiXTransferFunction _tf;
     bool _boundingBoxBuilt = false;
```

Comment:
Why _tf instead _transferFunction and _boundingBoxBuilt instead of _bbBuilt? Be consistent (in this case I prefer the longer names as tf is not a convention as Kd is).